explainer.md

```diff
@@ -86,39 +116,73 @@ Note that as part of this proposal, we are proposing to [remove the requirement]
-
-```js
-digitalGoodsService.acknowledge(purchaseToken, 'repeatable');
-```
+Some stores will require that the developer acknowledge a purchase once it has succeeded. In this case, the payment response will return a "purchase token" string, which should be verified and acknowledged by the developer. This should be done using direct API calls from the developer's server to the service provider before granting entitlements. Such a direct developer-to-provider API is not specified as part of the Digital Goods API.
 
-Items that are designed to be purchased once and last permanently in the user’s app must be acknowledged with the `onetime` flag. An example of a one-time purchase is a “remove ads” option. Once acknowledged with the `onetime` flag, the app is expected to remember the user’s purchase and continue providing the purchased capability.
+### Consuming a purchase
 
+Items that are designed to be purchased multiple times usually need to be marked as "consumed" before they can be purchased again by the user. An example of a consumable purchase is an in-game powerup that makes the player stronger for a short period of time. This can be done with the `consume` method:
 
 ```js
-digitalGoodsService.acknowledge(purchaseToken, 'onetime');
+digitalGoodsService.consume(purchaseToken);
 ```
 
-### Checking existing purchases
+It is preferable to use a direct developer-to-provider API to consume purchases, if one is available, in order to more verifiably ensure that a purchase was used up.
 
-The `listPurchases` method allows a client to get a list of items that are currently owned or purchased by the user. This may be necessary to check for entitlements (e.g. whether a subscription, promotional code, or permanent upgrade is active) or to recover from network interruptions during a purchase (e.g. item is purchased but not yet acknowledged).
+### Checking existing purchases
 
+The `listPurchases` method allows a client to get a list of items that are currently owned or purchased by the user. This may be necessary to check for entitlements (e.g. whether a subscription, promotional code, or permanent upgrade is active) or to recover from network interruptions during a purchase (e.g. item is purchased but not yet acknowledged). The method returns item IDs and purchase tokens, which should be verified using a direct developer-to-provider API before granting entitlements.
 
 ```js
 purchases = await digitalGoodsService.listPurchases();
 for (p of purchases) {
-  if (!p.acknowledged) {
-    await digitalGoodsService.acknowledge(p.purchaseToken, OnetimeOrRepeatable(p.itemId));
-    RecordSuccessfulPurchase(p);
-  }
-  DoSomethingWithEntitlement(p.itemId);
+  VerifyAndGrantEntitlement(p.itemId, p.purchaseToken);
 }
 ```
 
 ## Full API interface
 
+### API v2.0
+Available in Chrome M96 onwards.
```

Comment:
Feels questionable to put this detail in spec text.

Reply:
I agree... but is there a better place to put it for now? I think more developers are likely to refer to this than our I2E or Chromestatus.
I'll link to chromestatus and change to refer to OT.

Reply:
SGTM.